src/text.rs

```diff
@@ -367,7 +370,8 @@ impl TextContextImpl {
         face_index: u32,
     ) -> Result<FontId, ErrorKind> {
         self.clear_caches();
-
+        let mut data = data.as_ref().to_owned();
```

Comment:
This line ends up making a deep copy of the supplied font data. On my Ubuntu our gallery example with some text font-config quickly picks a 20MB NotoSans-CJK variant. Add bold text and an emoji and we're up to ~50MB of additional RAM usage (previously shared mmap).

I'm torn about this - I love this nicely encapsulated fix, but it pains to see memory usage go up. The other option is to downgrade dependencies (and for me that comes at the cost of bigger binary, but that's at least mmapped).

Either way it's temporary and either is fine with me ultimately - just wanted to add this data point :-)